Description: Fix cleanup rule in Makefile
 also remove generated vala files
Author: Rico Rommel <rico@bierrommel.de>
--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -88,4 +88,6 @@
 	libfso_glib_la.vala.stamp
 
 DISTCLEANFILES = \
-	Makefile.in
+	Makefile.in \
+	$(libfso_glib_la_VALASOURCES) \
+	freedesktop-gypsy.vala
